lib/CL/devices/remote/communication.c

```diff
@@ -1321,6 +1321,29 @@ traffic_monitor_pthread (void *arg)
   free (a);
   return NULL;
 }
+
+__attribute__ ((visibility ("default"))) void
```

Comment:
What's the attribute for?

Reply:
the rest of the file has the visibility hidden and this function is used by clients to get info about the amount of data being sent.

Reply:
The functions that are actually meant to be part of the ABI are defined in the header with POCL_EXPORT but I didn't want to advertise this function like that originally when it was just for some internal testing. If we want to integrate this more "officially" the function should be declared in the corresponding header with the appropriate macros.

Reply:
I already discussed the PoCL-specific (clGetDeviceInfo device query) extension with Yash. I think that's the clean way to go. Pls discuss with @Yashvardhan-A

Reply:
I've made the change by using clGetDeviceInfo